### PR TITLE
#156958153 Create New Reaction Theme

### PR DESCRIPTION
**Pull request by Efosaok**
#### What does this P.R do?
Create new reaction theme for the reaction-commerce app.

#### Description of task to be completed?
A new reaction theme should be created for the reaction commerce app, the email templates should reflect the new reaction theme.

#### snapshots
screen shot 2018-05-24 at 2 42 07 pm

#### Relevant pivotal tracker stories
#156958153

### CODE REVIEW

File: e2eTest/testButtonRateShop.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/* global module */
```
Comment (morekoya) [inline e2eTest/testButtonRateShop.js]: Why is this here? It has nothing to do with the team.

File: imports/plugins/core/checkout/client/components/cartItems.js [hunk 2]
```diff
@@ -53,8 +53,8 @@ class CartItems extends Component {
             <div className="center-cropped" style={{ backgroundImage: `url(${handleImage(item).url({ store: "small" })})` }}>
               <img src={handleImage(item).url({ store: "small" })} className="product-grid-item-images img-responsive" />
             </div> :
-            <div className="center-cropped" style={{ backgroundImage: "url('/resources/placeholder.gif')" }}>
-              <img src="/resources/placeholder.gif" className="product-grid-item-images img-responsive" />
+            <div className="center-cropped" style={{ backgroundImage: "url('/resources/sneaker1.jpeg')" }}>
```
Comment (morekoya) [inline imports/plugins/core/checkout/client/components/cartItems.js:56]: Should this be hardcoded?

File: imports/plugins/included/default-theme/client/styles/buttonSelect.less [hunk 3]
```diff
@@ -54,9 +54,9 @@
     text-align: center;
 }
 
-.button-select .button-dropdown .button-item:hover {
-    background: #f5f5f5;
-}
+// .button-select .button-dropdown .button-item:hover {
```
Comment (morekoya) [inline imports/plugins/included/default-theme/client/styles/buttonSelect.less]: Is this still being used?

File: imports/plugins/included/product-detail-simple/client/components/addToCartButton.js [hunk 4]
```diff
@@ -16,16 +16,22 @@ class AddToCartButton extends Component {
...
+              <label>Select quantity:</label>
+              <input
+                className="form-control input-md quantity-e"
+                id="add-to-cart-quantity"
+                min="1"
+                name="addToCartQty"
+                onChange={this.handleCartQuantityChange}
+                type="number"
+                value={this.props.cartQuantity}
+                placeholder="Qty"
```
Comment (morekoya) [inline imports/plugins/included/product-detail-simple/client/components/addToCartButton.js]: Won't it be better to have this written in full?

File: imports/plugins/included/product-variant/components/productGrid.js [hunk 5]
```diff
@@ -32,6 +32,9 @@ class ProductGrid extends Component {
   render() {
     return (
       <div className="container-main">
+        {/* <div className="slide-shows">
```
Comment (morekoya) [inline imports/plugins/included/product-variant/components/productGrid.js]: I believe this can be taken out if it is no longer in use?

File: imports/plugins/included/ui-search/lib/components/searchModal.js [hunk 6]
```diff
@@ -24,7 +24,7 @@ class SearchModal extends Component {
     return (
       <div className="rui search-modal-input">
         <label data-i18n="search.searchInputLabel">Search {this.props.siteName}</label>
-        <i className="fa fa-search search-icon" />
+        <i className="fa fa-search seach-side search-icon" />
```
Comment (morekoya) [inline imports/plugins/included/ui-search/lib/components/searchModal.js]: I believe `seach-side` is a typo. Please correct it.

File: reports/CHROME_66.0.3359.139_Mac OS X_adminTour.test.xml [hunk 7]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (morekoya) [inline reports/CHROME_66.0.3359.139_Mac OS X_adminTour.test.xml]: I believe test reports do not need to be pushed to the repo. Please add the folder to gitignore

File: imports/plugins/included/product-detail-simple/client/components/addToCartButton.js (hunk 4 above)
Comment (Efosaok) [reply imports/plugins/included/product-detail-simple/client/components/addToCartButton.js]: the placeholder value does not actually have any effect, so I'll remove it..

File: imports/plugins/core/checkout/client/components/cartItems.js (hunk 2 above)
Comment (Efosaok) [reply imports/plugins/core/checkout/client/components/cartItems.js:56]: Oh, Yes It has to be hardcoded, that is where the images for the app is stored, images are not stored in the database

**Comment by morekoya** [comment]
When you pull, do you use "git pull -r"?

File: imports/plugins/included/ui-search/lib/components/searchModal.js (hunk 6 above)
Comment (Efosaok) [reply imports/plugins/included/ui-search/lib/components/searchModal.js]: I did not work on this file as well, this is as a result of pulling from develop, so I am not sure if fixing the typo would affect other things

File: imports/plugins/included/product-variant/components/productGrid.js (hunk 5 above)
Comment (Efosaok) [reply imports/plugins/included/product-variant/components/productGrid.js]: ok, I'll do that

File: imports/plugins/included/default-theme/client/styles/buttonSelect.less (hunk 3 above)
Comment (Efosaok) [reply imports/plugins/included/default-theme/client/styles/buttonSelect.less]: no, removing it..

File: e2eTest/testButtonRateShop.js (hunk 1 above)
Comment (Efosaok) [reply e2eTest/testButtonRateShop.js]: I'll remove it now..

File: reports/CHROME_66.0.3359.139_Mac OS X_adminTour.test.xml (hunk 7 above)
Comment (Efosaok) [reply reports/CHROME_66.0.3359.139_Mac OS X_adminTour.test.xml]: It has been added to the gitignore file before now initially, but I'll look into it now..

File: imports/plugins/core/checkout/client/components/completedOrder.js [hunk 8]
```diff
@@ -69,7 +69,9 @@ const CompletedOrder = ({ order, orderId, shops, orderSummary, paymentMethods, h
       </div>
 
       <div className="order-details-side">
-
+        <div>
+          <button className="btn btn-default btn-rate-shop btn-block">RATE THIS SHOP</button>
```
Comment (morekoya) [inline imports/plugins/core/checkout/client/components/completedOrder.js]: It seems like you have mixed in elements of your other PR.

File: imports/plugins/included/product-detail-simple/client/components/addToCartButton.js [hunk 9]
```diff
@@ -16,16 +16,22 @@ class AddToCartButton extends Component {
...
+              <label>Select quantity:</label>
+              <input
+                className="form-control input-md quantity-e"
+                id="add-to-cart-quantity"
+                min="1"
+                name="addToCartQty"
+                onChange={this.handleCartQuantityChange}
+                type="number"
+                value={this.props.cartQuantity}
+                placeholder="quantitiy"
```
Comment (morekoya) [inline imports/plugins/included/product-detail-simple/client/components/addToCartButton.js]: Please pay more attention to detail

**Comment by iidrees** [comment]
@Efosaok another merge conflict to be resolved.
